### ISSUE TITLE
[New App Shortcut]: ITVX

### ISSUE THREAD

**Issue by DJBenson** [PRProd/HA-Firemote#346]
### App Name

ITVX

### Device Platform

Amazon Fire

### App Store Link and/or URL to .apk file if sideloaded

https://www.amazon.co.uk/ITV-Hub/dp/B00PH7XGQG

### Application Details
```
  mSurface=Surface(name=air.ITVMobilePlayer/air.ITVMobilePlayer.ITVActivity)/@0xc6087ef
  mCurrentFocus=Window{9af672d u0 air.ITVMobilePlayer/air.ITVMobilePlayer.ITVActivity}
  mFocusedApp=AppWindowToken{1f1835e token=Token{745c499 ActivityRecord{707a9e0 u0 air.ITVMobilePlayer/.ITVActivity t33}}}
```

**Comment by PRProd** [comment]
Greetings @DJBenson!

I've just published Firemote beta version [v3.3.0b2](https://github.com/PRProd/HA-Firemote/releases/tag/v3.3.0b2) which contains your request.  I was unable to test this button with my Amazon Fire devices (regional restrictions) so I'd like for you to download the beta and let me know if it works properly or not.  Thanks for your help!

-Doug

**Comment by DJBenson** [comment]
Will do, many thanks!!

**Comment by PRProd** [comment]
@DJBenson 

I've just published Firemote version [v3.3.0](https://github.com/PRProd/HA-Firemote/releases/tag/v3.3.0) which contains the same launcher code as the beta did.  Please let me know if this doesn't work as expected, but if it does, enjoy!

-Doug
